modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio;
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( Link_In_Bio $widget ): void {
+		?>
+		<div class="e-link-in-bio">
+			<div class="e-link-in-bio__content">
+
+				<?php $this->render_identity_image( $widget ); ?>
+
+				<?php $this->render_bio( $widget ); ?>
+
+				<?php $this->render_icons( $widget ); ?>
+
+				<?php $this->render_ctas( $widget ); ?>
+
+			</div>
+			<div class="e-link-in-bio__bg">
+				<div class="e-link-in-bio__bg-overlay"></div>
+			</div>
+		</div>
+		<?php
+	}
+
+	private function render_ctas( Link_In_Bio $widget ): void {
+		$settings               = $widget->get_settings_for_display();
+		$ctas_props_corners     = $settings['cta_links_corners'] ?? 'rounded';
+		$ctas_props_show_border = $settings['cta_links_show_border'] ?? false;
+		$ctas_props_type        = $settings['cta_links_type'] ?? 'button';
+		$ctas_value             = $settings['cta_link'] ?? [];
+
+		$has_ctas = ! empty( $ctas_value );
+		if ( ! $has_ctas ) {
+			return;
+		}
+
+		$widget->add_render_attribute( "ctas", [
+			'class' => "e-link-in-bio__ctas has-type-{$ctas_props_type}",
+		] );
+		?>
+
+		<div <?php echo $widget->get_render_attribute_string( "ctas" ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
+			<?php
+			foreach ( $ctas_value as $key => $cta ) {
+				// Bail if no text
+				if ( empty( $cta['cta_link_text'] ) ) {
+					break;
+				}
+				$formatted_link = $this->get_formatted_link_based_on_type_for_cta( $cta );
+
+				// Bail if no link
+				if ( empty( $formatted_link ) ) {
+					break;
+				}
+
+				$ctas_classnames = "e-link-in-bio__cta is-type-{$ctas_props_type}";
```

Comment:
```suggestion
				$ctas_classnames = 'e-link-in-bio__cta is-type-' . $ctas_props_type;
```
@nicoladj77 Please check this everywhere.